docs/gcm.md

```diff
@@ -1,3 +1,24 @@
 ## Running Heapster in Kubernetes with a Google Cloud Monitoring backend
 
-**Coming soon!**
+This guide assumes you are running on Google Compute Engine.
+
+### Setup a Kubernetes cluster
+[Bring up a Kubernetes cluster](https://github.com/GoogleCloudPlatform/kubernetes), if you haven't already. Ensure that `kubecfg.sh` is exported. By default, [cAdvisor](https://github.com/google/cadvisor) runs on all nodes in port 4194.
```

Comment:
We don't access cadvisor directly anymore in heapster. Should we even mention cadvisor port here?